vpr/src/base/vpr_context.h

```diff
@@ -326,9 +326,9 @@ struct ClusteringHelperContext : public Context {
     // the utilization of external input/output pins during packing (between 0 and 1)
     t_ext_pin_util_targets target_external_pin_util;
 
-    ~ClusteringHelperContext() {
-        delete[] primitives_list;
-    }
+    //~ClusteringHelperContext() {
```

Comment:
Better to delete this code rather than commenting it out if it isn't needed anymore.